mnd/mnd/migrations/0017_primary_carer_email_unique_data_migration.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from collections import defaultdict
+import json
+from django.db import migrations, models
+
+
+
+def primary_carer_unique_email(apps, schema_editor):
```

Comment:
Do we need this migration? What does django normally do when you make an existing field unique?

If the field wasn't already functionally unique, then I imagine there would be a runtime error from postgres. But can't we simplify this migration by manually removing non-unique carers (if there are any)?

If there were carers in production then I'd say we should do it properly, and I'm happy to do it this way if it's a good idea.

Reply:
This was implemented assuming there are non-unique emails in production. 
The field wasn't already functionally unique as you could theoretically add two carers with the same email address ("email1@test.com" and "Email1@test.com" were treated as different addresses).
I still think It's a good idea to use a migration as it's more explicit and it's less prone to error than having to manually fix entries in the db.